maintenance/errors/error.go

```diff
@@ -158,7 +161,8 @@ type sentryEvent struct {
 }
 
 func (e sentryEvent) Send() {
-	raven.Capture(e.build(), nil)
+	_, errCh := raven.Capture(e.build(), nil)
+	<-errCh
```

Comment:
Debugging artifact?

Reply:
No, intended. This change ensures that the sentry message gets send even if the process itself get's killed.

Reply:
Oh I see. Can you add that as a comment to the code, please?